db/migrate/20150714144500_review_discovery_permissions.rb

```diff
@@ -0,0 +1,12 @@
+class ReviewDiscoveryPermissions < ActiveRecord::Migration
+  def up
+    if mgr = Role.find_by_name("Discovery Manager")
+      mgr.add_permissions!(["submit_discovered_hosts", "auto_provision_discovered_hosts", "create_discovery_rules"])
```

Comment:
maybe only add these if the `provision_discovered_hosts` permission exists since these three replace it and we wouldn't want to readd permissions if someone deliberately removed them